### PR TITLE
Add docs for metric tables

### PR DESCRIPTION
**Pull request by alexander-gr**
## Changes

Add  metric tables docs.

TODO: add link to label_fqn page once it's done

## Checklist

- [x] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

**Comment by alexander-gr** [comment]
@borisvuchkov Do you approve moving `Using precomputed metrics to build new metrics` section from `writting sql queries` page to `metric tables` page?

File: src/docs/santiment-queries/metric-tables/index.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+
+This document introduces Santiment's precomputed metrics datasets.
+
+You can filter all metric tables as:
+
+```sql
+SHOW TABLES LIKE '%metric%'
+```
+
+Test in [Queries](https://queries.santiment.net/query/list-of-tables-390)
```
Comment (IvanIvanoff) [inline src/docs/santiment-queries/metric-tables/index.md:18]: The example shows tables with `like '%price%'`

File: src/docs/santiment-queries/metric-tables/index.md [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+
+Usually, tables storing pre-computed data have a common set of columns:
+
+- `dt` - A `DateTime` field storing the corresponding date and time.
+- `asset_id` - An `UInt64` unique identifier for an asset. The data for that ID is stored in the `asset_metadata` table.
+- `metric_id` - An `UInt64` unique identifier for a metric. The data for that ID is stored in the `metric_metadata` table.
+- `value` - A `Float` column holding the metric's value for the given asset/metric pair.
+- `computed_at` - A `DateTime` column storing the date and time when the given row was computed.
+
+Metrics are located in a different tables based on different parameters. All tables are similar to each other and described below.
```
Comment (IvanIvanoff) [inline src/docs/santiment-queries/metric-tables/index.md:28]:
```suggestion
Metrics are located in different tables, depending on some different parameters. All tables are similar to each other and described below.
```
We should give examples. I was about to say that `intraday` contains intraday metrics and `daily` contains daily metrics, but there are a lot of daily metrics in intraday.

File: src/docs/santiment-queries/metric-tables/index.md [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+
+## Tables
+
+---
+
+### daily_metrics_v2
+
+Table `daily_metrics_v2` is the most basic table. It stores a signle metric value for each asset for each day. I.e. in that tables stored metrics that have exactly 1 value per day.
```
Comment (IvanIvanoff) [inline src/docs/santiment-queries/metric-tables/index.md:36]:
```suggestion
The `daily_metrics_v2` table stores a single metric value for each asset for each day. I.e. in that tables stored metrics that have exactly 1 value per day.
```

File: src/docs/santiment-queries/metric-tables/index.md [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+> Note that this table uses old way to filter labels (plain label/owner instead of label_fqn).
+
+This table could be used to retrieve [labelled exchange](/metrics/labeled-exchange) metrics under names: 
+
+* `labelled_active_deposits`
+* `labelled_deposit_transactions`
+* `labelled_active_withdrawals`
+* `labelled_withdrawal_transactions` 
+
+or NFT [trading volume](metrics/nft-trade-volume-usd) and [trades count](metrics/nft-trades-count) metrics.
```
Comment (borisvuchkov) [inline src/docs/santiment-queries/metric-tables/index.md:120]: Those links doesn't work. Maybe the metric has been renamed or removed?

Comment (alexander-gr) [reply src/docs/santiment-queries/metric-tables/index.md:120]: Should work once in master, Kate added these articles in a separate branch https://github.com/santiment/academy/pull/393

File: src/docs/santiment-queries/metric-tables/index.md [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+---
+
+## Overview
+
+This document introduces Santiment's precomputed metrics datasets.
+
+You can filter all metric tables as:
+
+```sql
+SHOW TABLES LIKE '%metric%'
```
[the remark this answers is not in the document]
Comment (alexander-gr) [reply src/docs/santiment-queries/metric-tables/index.md:15]: I dont think this is a issue, isn't it?

File: src/docs/santiment-queries/metric-tables/index.md (hunk 1 above)
Comment (alexander-gr) [reply src/docs/santiment-queries/metric-tables/index.md:18]: Fixed

File: src/docs/santiment-queries/metric-tables/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (alexander-gr) [reply src/docs/santiment-queries/metric-tables/index.md]: Anyway I've changed it into `%metrics%` so metric_metadata is not shown

**Pull request merged**